rcsb/utils/chemref/RcsbLigandReferenceGenerator.py

```diff
@@ -0,0 +1,300 @@
+##
+#  File:           RcsbLigandReferenceGenerator.py
+#  Date:           2025-12-10 Chenghua Shao
+#
+#  Update:
+##
+"""
+Generate ligand quality reference data that is used for ligand quality score computation.
+
+"""
+import csv
+import logging
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+from rcsbapi.data import DataQuery as Query
+from rcsbapi.data import ALL_STRUCTURES
+from rcsbapi.config import config
+from rcsb.utils.io.StashableBase import StashableBase
+
+logger = logging.getLogger(__name__)
+
+
+class RcsbLigandReferenceGenerator(StashableBase):
```

Comment:
I don't think you need `StashableBase` here. Once I figure out how to manage the stashing process, we can think about adding it back in.